template/build.gradle.kts

```diff
@@ -26,13 +26,18 @@ allprojects {
     }
 }
 
+configurations.all {
+    resolutionStrategy.cacheChangingModulesFor(0, TimeUnit.SECONDS)
+    resolutionStrategy.cacheDynamicVersionsFor (0, TimeUnit.SECONDS)
+}
+
 tasks.register("clean", Delete::class) {
     delete(rootProject.buildDir)
 }
 
 tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
     // Target version of the generated JVM bytecode. It is used for type resolution.
-    jvmTarget = JavaVersion.VERSION_1_8.toString()
+    jvmTarget = JavaVersion.VERSION_11.toString()
 }
```

Comment:
Could it be this is redundant? Since we already define `tasks.withType<io.gitlab.arturbosch.detekt.Detekt>` on line 64

Reply:
@Tuubz Good catch, this is the redundant one 🙏